src/main/java/frc/robot/subsystems/Conveyor.java

```diff
@@ -0,0 +1,148 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import frc.robot.utilities.Functions;
+import frc.robot.utilities.lists.Ports;
+
+/**
+ * Subsystem to control the conveyor of the robot.
+ */
+public class Conveyor extends SubsystemBase {
+
+    // motors
+    private final CANSparkMax front = new CANSparkMax(Ports.FRONT_CONVEYOR, MotorType.kBrushless);
+    private final CANSparkMax back = new CANSparkMax(Ports.BACK_CONVEYOR, MotorType.kBrushless);
+
+    // encoders
+    private final RelativeEncoder frontEncoder = front.getEncoder();
+    private final RelativeEncoder backEncoder = back.getEncoder();
+
+    public Conveyor() {
+        zeroEncoders();
+    }
+
+    /**
+     * Sets the power of the front motor.
+     * 
+     * @param power
+     */
+    public void setFrontMotorPower(double power) {
+        power = Functions.clampDouble(power, 1.0, -1.0);
```

Comment:
clamping the power btween 1 and -1 is auctualy unnessary, the call to set() will do that its self